Doc/library/tkinter.rst

```diff
@@ -67,13 +67,33 @@ Or, more often::
    from tkinter import *
 
 
-.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1)
-
-   The :class:`Tk` class is instantiated without arguments. This creates a toplevel
-   widget of Tk which usually is the main window of an application. Each instance
-   has its own associated Tcl interpreter.
-
-   .. FIXME: The following keyword arguments are currently recognized:
+.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1, sync=0, use=None)
+
+   Return a toplevel Tk widget, which is usually the main window of an
+   application.  Each instance has its own associated Tcl interpreter.
+
+   The :class:`Tk` class is typically instantiated using all default values.
+   However, the following keyword arguments are currently recognized:
+
+   *screenName*
+      Sets the DISPLAY environmental variable. (X11 only)
+   *baseName*
+      Reads the profile file :file:`baseName.tcl` into the Tcl interpreter and
```

Comment:
```rst
:file:`{baseName}.tcl`
```

Reply:
How about "When given, read ... and call ... ."

Reply:
The profile is read even if *baseName* is not given. By default *baseName* is derived from the program name (`sys.argv[0]`).